### PR TITLE
fix reflame antd style discrepancies

### PR DESCRIPTION
**Pull request by Vadman97**
## Summary

Fixes additional antd override discrepancies in reflame deploys.

## How did you test this change?

before
Screenshot from 2023-11-30 18-21-16

after
Screenshot from 2023-11-30 18-21-21

before
Screenshot from 2023-11-30 18-20-44

after
Screenshot from 2023-11-30 18-20-49

## Are there any deployment considerations?

No

## Does this work require review from our design team?

No

### CODE REVIEW

File: frontend/src/index.tsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import './__generated/antd.css'
 import '@highlight-run/rrweb/dist/rrweb.min.css'
 import '@fontsource/poppins'
 import '@highlight-run/ui/styles.css'
 import './index.css'
+import './__generated/antd.css'
```
Comment (lewisl9029) [inline frontend/src/index.tsx:5]: There might not be any practical implications looking at the preview, but I think it makes more sense for our custom css to go after the generated antd.css so they can override them when necessary.

Comment (Vadman97) [reply frontend/src/index.tsx:5]: oops yeah, was trying that - will revert!

File: frontend/vite.config.ts [hunk 2]
```diff
@@ -108,6 +112,17 @@ export default defineConfig(({ mode }) => {
...
+				less: {
+					javascriptEnabled: true,
+					modifyVars: {
+						hack: `true; @import "${join(
+							__dirname,
+							'src/style/AntDesign/antd.overrides.less',
+						)}";`,
+					},
+				},
+			},
```
Comment (Vadman97) [inline frontend/vite.config.ts:125]: @lewisl9029 i put this all back because vite dev locally would not have antd styling otherwise. should be ok, right?

Comment (Vadman97) [reply frontend/vite.config.ts:125]: yeah, reflame preview is good without this change (isn't affected by vite.config.ts), but `vite dev` didn't have the antd styling without both of these changes in the vite.config.ts

Comment (Vadman97) [reply frontend/vite.config.ts:125]: Screenshot from 2023-11-30 19-07-00

hmmm

Comment (Vadman97) [reply frontend/vite.config.ts:125]: this is actually how main looks right now for me, running with `./run-frontend.sh`

Comment (lewisl9029) [reply frontend/vite.config.ts:125]: Sorry took a while, but finally managed to repro. Had to comment out [these lines](https://github.com/lewisl9029/highlight/blob/ab02ba0554ffa62385cec6171d9b9aca82a087e6/docker/env.sh#L41-L48) for run-backend.sh to work properly after running start-infra.sh

Looking into it now.

Comment (lewisl9029) [reply frontend/vite.config.ts:125]: Wow, turns out, vite-plugin-imp has a preset option for antd to add imports to their css entry points:
Screenshot 2023-11-30 at 8 31 22 PM

And these overwrote classes from our prebuilt css. 🤦 

Looking at the list of libraries with built-in support, I don't think we needed it for anything other than antd previously, and we no longer need it for antd thanks to the prebuilt stylesheet, so I ended up just removing it altogether in https://github.com/highlight/highlight/pull/7255/commits/ebb92c7f65db0182958e2d7e289de40f13c2790b. 

Longer term we might add something similar natively in Reflame to help users map named imports to specific entry points automatically to better optimize their prod bundles.

Comment (Vadman97) [reply frontend/vite.config.ts]: wow, interesting! thanks for getting to the bottom of this. looks good with your changes locally, merging!

**Pull request merged**